src/app/docs/guides/voice-chat/page.md

```diff
@@ -18,6 +18,42 @@ A detailed video and step by step instructions are coming soon, in the meantime,
 we've called out the important parts of the repo above to help you get started
 with it.
 
+## Quickstart to Running the Sample App
+
+```shell
+git clone https://github.com/sublayerapp/rails_llm_voice_chat_example.git
+```
+
+```shell
+bundle install
+```
+
+```shell
+rails db:migrate
+```
+
+```shell
+export OPENAI_API_KEY=your_openai_api_key` 
+```
+
+<!-- should we use something like `.env` instead? -->
+
+```shell
+rails s
+```
+
+[http://127.0.0.1:3000/](http://127.0.0.1:3000/)
+Now hold the button and talk to the chat bot 😄
+
+```shell
```

Comment:
What's your thinking here? Just a way to browse your conversation?

Reply:
it was the answer to the immediate next question I had, "where does this app keep transcripts?"